server/views/pages/visit/confirmUpdate.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
```

Comment:
Don't need `govukSummaryList` - and surprised it works with out the `govukRadios` import 🤔!